Examples/Scripts/PostFreeSurferPipelineBatch-long.sh

```diff
@@ -0,0 +1,243 @@
+#!/bin/bash 
+
+get_usage_and_exit(){
+    echo "usage: "
+    echo "PostFreeSurferPipelineBatch-long.sh [options]"
+    echo "options:"
+    echo "  --runlocal                      run locally [FALSE]"
+    exit -1
+}
+
+command_line_specified_run_local=FALSE
+while [ -n "$1" ]; do
+    case "$1" in
+        --runlocal) shift; command_line_specified_run_local=TRUE ;;
+        *) shift ;;
+    esac
+done
+
+#################################################################################################
+# General input variables
+##################################################################################################
+StudyFolder=/media/myelin/brainmappers/Connectome_Project/MishaLongitudinal/hcp/FSLong-test
+#StudyFolder="${HOME}/projects/Pipelines_ExampleData" #Location of Subject folders (named by subjectID)
+#list of subject labels, space separated
+Subjlist="HCA6002236"
+#list of timepoints per each subject. Subject timepoint groups are space separated, timepoints within group are @ separated
+Timepoint_list="HCA6002236_V1_MR@HCA6002236_V2_MR"
+#list of longitudinal template labels, one per subject, space separated
+Template_list="HCA6002236_V1_V2"
+#EnvironmentScript="${HOME}/projects/Pipelines/Examples/Scripts/SetUpHCPPipeline.sh" #Pipeline environment script
```

Comment:
We should use `HCPpipelines` rather than `Pipelines`, to reflect the name of the repo for years at this point.

Reply:
I agree but that's basically standard example path used across 26 script files. Should I include this change in all relevant files?

Reply:
Maybe we could do that as a simple, separate "clean-up" PR once this PR is done.

Reply:
I changed these paths in scripts that I created or modified.